owncloudApp/src/main/res/values/setup.xml

```diff
@@ -60,6 +60,10 @@
 
     <!-- Help, imprint and feedback -->
     <bool name="help_enabled">true</bool>
+
+
+
```

Comment:
There is no need to change anything. This is enough

`git commit -m "something" --allow-empty`